### PR TITLE
Update interface

### PR DESCRIPTION
**Pull request by ruihildt**

### CODE REVIEW

File: src/components/ProxyDetails/ProxyConnect.vue [hunk 1]
```diff
@@ -18,8 +18,9 @@ const showProxyButton = asyncComputed(() => extension.isAllowedIncognitoAccess()
     <ol class="ml-4 list-decimal">
       <li>
         <p class="flex items-center">
-          Click the&nbsp;<em class="inline-flex"><IcRoundMenu /></em>&nbsp;icon in the top right
-          corner of the browser
+          Click the
+          <FeMenu />
```
Comment (hankolsen) [inline src/components/ProxyDetails/ProxyConnect.vue:22]: You need a non-breaking space between `the` and the icon.
image

File: src/components/PrivacyRecommendations/Instructions.vue [hunk 2]
```diff
@@ -18,8 +18,9 @@ const title = computed(() => {
     <ol class="ml-4 list-decimal">
       <li>
         <p class="flex items-center">
-          Click the&nbsp;<em class="inline-flex"><IcRoundMenu /></em>&nbsp;icon in the top right
-          corner of the browser
+          Click the
```
Comment (hankolsen) [inline src/components/PrivacyRecommendations/Instructions.vue:21]: You need a non-breaking space here too 
image

**Review by hankolsen — APPROVED** [review]

File: src/components/ProxyDetails/ProxyConnect.vue (hunk 1 above)
Comment (ruihildt) [reply src/components/ProxyDetails/ProxyConnect.vue]: Fixed.

File: src/components/PrivacyRecommendations/Instructions.vue (hunk 2 above)
Comment (ruihildt) [reply src/components/PrivacyRecommendations/Instructions.vue]: Fixed.

**Pull request merged**